index.html

```diff
@@ -0,0 +1,149 @@
+              <li><i class="fab fa-facebook-f"></i></li>
+              <li><i class="fab fa-twitter"></i></li>
+              <li><a href=""> English</a></li>
+              <li><a href="">My Page</a></li>
+              <li><a href="">Logout</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <!-- Hamburger Icons for Mobile Menu -->
+      <div class="desktop-hidden container-fluid">
+        <i id="hamburger-menu" class="menu-icon-mobile fa fa-bars f a-lg fa-2x"></i>
+        <i id="close-menu-icon" class="menu-hidden menu-icon-mobile fas fa-times fa-2x"></i>
+      </div>
+
+      <!-- Links to Show on Mobile Menu -->
+      <div id="mobile-menu-links" class="menu-hidden mobile-menu-list text-center">
+        <div class="desktop-main-nav-links">
+          <ul class="p-3">
+            <li class="mobile-hidden"> <a href=""> <img class="img-fluid header-desktop-image-logo" src="./images/nkwashi-logo.png" alt=""></a> </li> 
+            <li> <a href="./about.html">About</a> </li>
+            <li> <a href="#program">Program</a> </li>
+            <li> <a href="#join-conference">Join</a> </li>
+            <li> <a href="#">Sponsor</a> </li>
+            <li> <a href="#">News</a> </li>
+          </ul>
+        </div>
+      </div>
+
+      <section id="program" class="headline-text">
+        <p class="greeting-text mb-1">"Hello, ba Lusaka!"</p>
+        <h1>NKWASHI CREATIVE & TECHNOLOGICAL SUMMIT 2021</h1>
+        <p class="conference-description-text">A joyful celebration believeing in the value of openess, African culture and sharing. 
+            Trying to create a positive change with people from all over the country trying to effect actual change in our 
+            communities! It is taking place in Nkwashi, Lusaka, Zambia!
+        </p>
+        <p class="header-time-text pt-4"> 2021.10.7(THU) ~ 8(FRI) </p>
+        <p class="header-place-text"> @ Chena Art House, Nkwashi, Lusaka! </p>
+      </section>
+    </header>
+
+    <main>
+    <section class="main-program">
+      <div class="container-fluid">
+        <h2> Main Program </h2>
+        <hr class="orange-dividing-line-program mx-auto">
+          <div class="desktop-outline-grid-program-container">
+          <div class="grid-program-container">
+            <img src="./images/lecture.png" class=" program-image-icons" alt="illustration showing lecturer presenting">
+            <div class="program-description-title">
+              Lecture
+            </div>
+            <div class="program-description-text">
+              Listen to the speakers from various countries about the messages on community building and innovation
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/exhibition.png" class="program-image-icons" alt="illustration showing an exhibition">
+            <div class="program-description-title">
+              <p>CC Exhibition </p>
+            </div>
```

Comment:
- [ ] On tablet screen(768px), the contents in Main Program section overflows the container. Consider dividing items into more rows on tablet screen.
**Screenshot** 👇
<img src='https://user-images.githubusercontent.com/33323077/125158436-18dec380-e169-11eb-8ce1-4706d6d13961.png' width='300px'/>